zeppelin-server/src/main/java/org/apache/zeppelin/rest/InterpreterRestApi.java

```diff
@@ -243,17 +244,26 @@ public Response addRepository(String message) {
   /**
    * get the metainfo property value
    */
-  @GET
+  @POST
```

Comment:
I wanted to send more amount of parameter data , hence started using `POST`
https://github.com/apache/zeppelin/pull/1678/files/6ec58e8ebf2493e5f417d5ee3c352ab3c4c793bd#diff-79927cd87b606e2efc3267cf436aa704R643
Also read this in one of the posts.
```
an important consideration when using GET for AJAX requests is that some browsers - IE in particular - will cache the results of a GET request
```
Not very sure if its applicable.
One another reason is to keep the url cleaner :)